WooCommerce/Classes/ViewRelated/Orders/Order Creation/CustomerSection/CustomerSearchUICommand.swift

```diff
@@ -0,0 +1,80 @@
+import Foundation
+import Yosemite
+
+/// Implementation of `SearchUICommand` for Customer search.
+///
+final class CustomerSearchUICommand: SearchUICommand {
+
+    typealias Model = Customer
+    typealias CellViewModel = TitleAndSubtitleAndStatusTableViewCell.ViewModel
+    typealias ResultsControllerModel = StorageCustomer
+
+    var searchBarPlaceholder: String = Localization.searchBarPlaceHolder
+
+    var searchBarAccessibilityIdentifier: String = "customer-search-screen-search-field"
+
+    var cancelButtonAccessibilityIdentifier: String = "customer-search-screen-cancel-button"
+
+    var resynchronizeModels: (() -> Void) = {}
+
+    var onDidSelectSearchResult: ((Customer) -> Void)
+
+    private let siteID: Int64
+
+    init(siteID: Int64, onDidSelectSearchResult: @escaping ((Customer) -> Void)) {
+        self.siteID = siteID
+        self.onDidSelectSearchResult = onDidSelectSearchResult
+    }
+
+    func createResultsController() -> ResultsController<StorageCustomer> {
+        let storageManager = ServiceLocator.storageManager
+        let descriptor = NSSortDescriptor(keyPath: \StorageCustomer.customerID, ascending: false)
+        return ResultsController<StorageCustomer>(storageManager: storageManager, sortedBy: [descriptor])
+    }
+
+    func createStarterViewController() -> UIViewController? {
+        nil
+    }
+
+    func createCellViewModel(model: Customer) -> TitleAndSubtitleAndStatusTableViewCell.ViewModel {
+        return CellViewModel(
+            id: "\(model.customerID)",
+            title: "\(model.firstName ?? "") \(model.lastName ?? "")",
+            subtitle: model.email,
+            accessibilityLabel: "",
```

Comment:
Accessibility Label is required here... see what it does in VoiceOver without it: it just announces:

> Button

Without any detail about the name of the user or anything else about the search result. 

If you're moving to another cell, you may not have to do anything, usually we get VoiceOver in cells for free, but in this case we are specific about what the label should be in the view model.